### PR TITLE
Integration and cleanup

### PR DESCRIPTION
**Pull request by pavan-nw**
Refactored backend and front end code to make it work seamlessly

### CODE REVIEW

File: backend/src/main/java/com/stock/admin/model/request/StockRequest.java [hunk 1]
```diff
@@ -14,8 +14,29 @@
     private Date stockDate;
     private int openingStock = 0;
     private int closingStock = 0;
+    private String shopCode;
+
```
Comment (pavan-nw) [inline backend/src/main/java/com/stock/admin/model/request/StockRequest.java]: NIT : There should not be no extra new lines

Comment (pavan-nw) [reply backend/src/main/java/com/stock/admin/model/request/StockRequest.java]: Apply to other places as well.

File: backend/src/main/java/com/stock/admin/model/request/StockSearchRequest.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    private String productName;
+    private String packaging;
+    @DateTimeFormat(pattern = "dd-MM-yyyy", iso = DateTimeFormat.ISO.DATE)
+    private Date stockDate;    
+    private String shopCode;
+    int pageNum=1;
+    int size=500;
+    String sortType="DESC";
+
```
Comment (pavan-nw) [inline backend/src/main/java/com/stock/admin/model/request/StockSearchRequest.java]: As mentioned above. No extra new lines should be added.

File: front-end/src/components/stocks/StocksFormHeader.tsx [hunk 3]
```diff
@@ -1,41 +1,49 @@
...
     setStockDate,
-} from '../../features/dailyStocks/actions';
+} from '../../features/stocks/actions';
+import { getShopCode } from '../../features/login/selectors';
 
-export const DailyStockHeader: React.FC = () => {
+export const StocksFormHeader: React.FC = () => {
     const dispatch = useDispatch();
     const products: Product[] | [] = useSelector(getProducts);
+    const uniqueProducts = products.filter((product, index) => {
```
Comment (pavan-nw) [inline front-end/src/components/stocks/StocksFormHeader.tsx]: Try to move this to utils

**Review by shreeharinw — APPROVED** [review]
Approving on behalf of Pavan

**Pull request merged**